src/timecode_test_parse.rs

```diff
@@ -84,7 +90,10 @@ mod test {
             frames: -86400,
             timecode: "-01:00:00:00".to_string(),
             runtime: "-01:00:03.6".to_string(),
-            feet_and_frames: "-5400+00".to_string(),
+            feet_and_frames_35mm_4perf: "-5400+00".to_string(),
```

Comment:
Awesome! Thanks for going through all these. It's super tedious but helps protect against breakages in the future.

I would really like these tests to assert on parsing as well as rendering. You can just add them to the `frames_sources` input `vec!` above like so:

```rust
        ParseCase{
            frames_sources: vec![
                Box::new("01:00:00:00".to_string()),
                Box::new("86400".to_string()),
                Box::new("5400+00".to_string()),
                Box::new(FeetFramesStr::new("5400+00", FilmFormat::FF35mm4perf)),
                Box::new(FeetFramesStr::new("4050+00.0", FilmFormat::FF35mm3perf)),
                Box::new(FeetFramesStr::new("2700+00", FilmFormat::FF35mm2perf)),
                Box::new(FeetFramesStr::new("4320+00", FilmFormat::FF16mm)),
                Box::new(86400i64),
                Box::new(86400u64),
                Box::new(86400i32),
                Box::new(86400u32),
                Box::new(86400isize),
                Box::new(86400usize),
            ],
```

Please make sure to test both the implicit 35mm, 4-perf input (what's currently there), and the explicit 35mm, 4perf version.